content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -266,6 +266,49 @@ visit [MCR Deployment Guide](https://docs.mirantis.com/mcr/20.10/install.html).
 Check the systemd unit named `cri-docker.socket` to find out the path to the CRI
 socket.
 
+## Overriding the sandbox (pause) image
```

Comment:
I wasn't sure if this should be it's own section or sub headings under the other CR docs, felt nice to keep this separate.

Reply:
I think these should be separate sub sections under the individual CRs.